dashboard/dashboard.py

```diff
@@ -1,12 +1,53 @@
+# pylint: disable=C0103
```

Comment:
What's this warning?

Reply:
Python doesn't like the file names when they're not snake case, but we need them to be capitalised for the dashboard.